app/views/invitations/new.html.haml

```diff
@@ -0,0 +1,24 @@
+%section.invite.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = bootstrap_form_for @invitation, class: 'invite' do |f|
+        %header
+          %h1 Invite a friend to join MyFlix!
+        %fieldset
+          .form-group
+            = f.label :friend_name, "Friend's Name"
+            .row
+              .col-sm-4
+                = f.text_field :friend_name, class: 'form-control', hide_label: true
```

Comment:
When you use `bootstrap_form_for` basically whole this:

```haml
          .form-group
            = f.label :friend_name, "Friend's Name"
            .row
              .col-sm-4
                = f.text_field :friend_name, class: 'form-control', hide_label: true
```

should be just `= f.text_field :friend_name, label: "Friend's Name"` and in the `bootstrap_form_for @invitation` you can pass in the `layout: :horizontal, control_col: "col-sm-4"` or you can change those col classes inline for each field. It takes a bit of trial and error and working with the documentation but in the end you could get rid of that bootstrap noise for simple forms like this :)